test/infrastructure/docker/Makefile

```diff
@@ -48,6 +48,10 @@ endif
 CONTROLLER_GEN := $(TOOLS_BIN_DIR)/controller-gen
 CONVERSION_GEN := $(TOOLS_BIN_DIR)/conversion-gen
 GOLANGCI_LINT := $(TOOLS_BIN_DIR)/golangci-lint
+KUSTOMIZE := $(abspath $(TOOLS_BIN_DIR)/kustomize)
```

Comment:
What about using abspath for all the binaries (this will be consistent with the root Makefile as well)